xtext-website/_posts/releasenotes/2020-12-01-version-2-24-0.md

```diff
@@ -12,6 +12,9 @@ Xtext 2.24.0 is a maintenance release.
 
 As you might have recognized, the number of people contributing to Xtext on a regular basis has declined over the past years and so has the number of contributions. At the same time the amount of work for basic maintenance has stayed the same or even increased with the new release cadence of Java and the Eclipse simultaneous release. Briefly: The future maintenance of Xtext is at risk. If you care, please join the discussion in [https://github.com/eclipse/xtext/issues/1721](https://github.com/eclipse/xtext/issues/1721).
 
+## Java based API for new Formatter
+
+With Xtext 2.24 we introduce a new experimental [API](https://github.com/eclipse/xtext-core/blob/master/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/AbstractJavaFormatter.java) that allows you to implement your Formatter(2) in Java instead of Xtend. We encourage you to give it a try and to give feedback.
```

Comment:
s/experimental/preliminary/
your Formatter -> the Formatter
in Java instead of Xtend -> in Java instead of Xtend. With Java11 and later, you can use local variable type inference and the equivalent of commonly used extensions are available via delegate methods.